exams/pearson/writers_test.py

```diff
@@ -186,7 +187,7 @@ def test_write_skips_invalid_rows(self):  # pylint: disable=no-self-use
 
 
 @ddt.ddt
-class CDDWriterTest(TSVWriterTestCase):
+class CDDWriterTest(TSVWriterTestCase, TestCase):
```

Comment:
I think since these tests mute the `post_save` signal that we never hit indexing code in these tests so it shouldn't be an issue.